tailwind.config.js

```diff
@@ -66,6 +66,7 @@ export default {
           20: '#dbe5ed',
           10: '#edf2f6',
         },
+        text: { tertiary: '#9898A0', secondary: '#56565E' },
```

Comment:
you do not need to add it as "text", because you will always wrote text-text-{color}. You should put your colors in "colors" key